src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/factory/enrollment/EnrollmentFactory.js

```diff
@@ -129,8 +134,24 @@ export class EnrollmentFactory {
     ) {
         // $FlowFixMe
         await cachedProgramTrackedEntityAttributes.asyncForEach(async (trackedEntityAttribute) => {
-            const element = await this.dataElementFactory.build(trackedEntityAttribute);
-            element && section.addElement(element);
+            if (trackedEntityAttribute?.id === 'plugin') {
```

Comment:
The string `'plugin'` is used a couple of times across the PR. Does it make sense to move to a const in DataEntryPlugin.const.js ?